modMcf/src/org/aion/mcf/valid/TransactionTypeRule.java

```diff
@@ -6,13 +6,18 @@
 import static org.aion.mcf.tx.TransactionTypes.FVM;
 import static org.aion.mcf.tx.TransactionTypes.FVM_CREATE_CODE;
 
+import com.google.common.annotations.VisibleForTesting;
+
 /**
  * Rules for validating transactions based on allowed types.
  *
  * @author Alexandra Roatis
  */
 public class TransactionTypeRule {
 
+    // allowing only balance transfers on AVM when this flag is equal to false.
+    private static boolean DEPLOY_AVM_CONTRACT_ALLOWED = false;
```

Comment:
Note that this will have to be set to **true** on the AVM test network branches.

Reply:
updated the settings for the avmtestnet.